fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/generator/DefaultNullInjectGenerator.java

```diff
@@ -55,19 +50,13 @@ public final class DefaultNullInjectGenerator implements NullInjectGenerator {
 	public static final List<String> DEFAULT_NOTNULL_ANNOTATION_TYPES = Collections.unmodifiableList(
 		Arrays.asList(
 			"javax.annotation.Nonnull",
-			"org.springframework.lang.NonNull",
-			"edu.umd.cs.findbugs.annotations.NonNull",
-			"org.jetbrains.annotations.NotNull",
-			"com.sun.istack.internal.NotNull",
-			"androidx.annotation.NonNull",
-			"android.support.annotation.NonNull",
+			"jakarta.annotation.Nonnull",
 			"javax.validation.constraints.NotNull",
+			"jakarta.validation.constraints.NotNull",
+			"org.springframework.lang.NonNull",
 			"lombok.NonNull",
```

Comment:
`lombok.NonNull` should be removed as well. 
https://github.com/projectlombok/lombok/blob/master/src/core/lombok/NonNull.java